packages/effect/dtslint/Number.tst.ts

```
@@ -0,0 +1,559 @@
+import type { Option } from "effect"
+import { HashSet, Number, pipe } from "effect"
```

Biome (1.9.4)

[error] Do not shadow the global "Number" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)